service/routes.py

```diff
@@ -439,14 +418,14 @@ def rename_a_wishlist(wishlist_id):
 ######################################################################
 # UPDATE A PRODUCT IN A WISHLIST
 ######################################################################
-@app.route("/wishlists/<int:wishlist_id>/products/<int:product_id>", methods=["PUT"])
-def update_product_in_wishlist(wishlist_id, product_id):
+@app.route("/wishlists/products/<int:product_id>", methods=["PUT"])
+def update_product_in_wishlist(product_id):
```

Comment:
And here. I'm sorry if I misled you by saying "delete by ID" only. I meant that we can call delete_by_id on a `Product` instance, not for the URL.